config/scout.php

```diff
@@ -75,6 +116,24 @@
         'secret' => env('ALGOLIA_SECRET', ''),
     ],
 
+    /*
+    |--------------------------------------------------------------------------
+    | MeiliSearch Configuration
+    |--------------------------------------------------------------------------
+    |
+    | Here you may configure your MeiliSearch settings. MeiliSearch is an open
+    | source search engine with minimal configuration. Below, you can state
+    | the host and key information for your own MeiliSearch installation.
+    |
+    | See: https://docs.meilisearch.com/guides/advanced_guides/configuration.html
+    |
+    */
+
+    'meilisearch' => [
+        'host' => env('MEILISEARCH_HOST', 'http://localhost:7700'),
+        'key' => env('MEILISEARCH_KEY', null),
```

Comment:
Do we need to add these to our local `.env`, or will these defaults work?

Reply:
We're not using Mielisearch, so we don't need these values in our .env files. Added this to config just to have parity with the version of the package we're using.